src/Components/TodoForm.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+
+function TodoForm({ addTodo }) {
+    const [value, setValue] = useState("");
+
+    const updateValue = e => {
+        setValue(e.currentTarget.value);
+    };
+
+    const handleSubmit = e => {
```

Comment:
Again, this should be passed down as prop and handled in the parent since it is dealing with the state which is up there. 

`updateValue` however, is a good example of where it would be appropriate to have it within this component. If we reuse this component, that is something that we would want encapsulated inside this  because its use case is here only.